Wikipedia/Code/WMFArticleContainerViewController+TOC.swift

```diff
@@ -50,7 +50,11 @@ extension WMFArticleViewController : WMFTableOfContentsViewControllerDelegate {
     }
 
     public func tableOfContentsArticleSite() -> MWKSite {
-        return self.articleTitle.site
+        if(self.articleTitle.isNonStandardTitle()){
+            return MWKSite.init(language: "en")
```

Comment:
Yeah. Basically. The only thing we are doing this for now is mediawiki.org sites. I don't like it but really we just need to rip out mwksite and mwktitle